analytics_platform_concourse_webhook_dispatcher/server.py

```diff
@@ -37,15 +40,27 @@
     route = events.get(app, event, body)
     if not route:
         return json({}, status=404)
-    url = concourse.webhook_url(
-        app.config.CONCOURSE_BASE_URL,
-        route["team"],
-        route["pipeline"],
-        route["resource"],
-        app.config.CONCOURSE_WEBHOOK_TOKEN,
+
+    concourse_url = app.config.CONCOURSE_BASE_URL
+
+    run_args = [
+        "check-resource",
+        "--resource", f"{route['pipeline']}/{route['resource']}"
+    ]
+
+    fly = Fly(
+        concourse_url=concourse_url
     )
-    print(url)
```

Comment:
this is pretty crappy on my part but the printing of the url was quite useful for observability... you could do it properly and add a bit of logging?